app/src/main/res/values-night-v29/themes.xml

```diff
@@ -15,17 +15,38 @@
         <!-- Customize your theme here. -->
         <item name="android:forceDarkAllowed">true</item>
 
-
     </style>
-
-    <style name="textTitle">
+    <style name="TextTitle">
         <item name="android:layout_width">match_parent</item>
         <item name="android:layout_height">wrap_content</item>
         <item name="android:fontFamily">@font/roboto_medium</item>
         <item name="android:textColor">@color/txt_white_dark_background</item>
         <item name="android:textSize">@dimen/textXLarge</item>
     </style>
-
+    <style name="TextKitchens">
+        <item name="android:layout_marginStart">@dimen/spaceMedium</item>
+        <item name="android:layout_marginBottom">@dimen/spaceSmall</item>
+        <item name="fontFamily">@font/roboto_bold</item>
+        <item name="android:textSize">@dimen/textXLarge</item>
+        <item name="android:textColor">@color/white</item>
+    </style>
+    <style name="TextFood">
+        <item name="android:layout_marginHorizontal">@dimen/spaceSmall</item>
```

Comment:
use android:layout_marginStart and android:layout_marginEnd instead of android:layout_marginHorizontal